core-skills/tdd/README.md

```diff
@@ -24,91 +24,104 @@
 
 ## Mark Scheme
 
-In combination with [madetech/core-skill-tdd](https://github.com/madetech/core-skill-tdd) repository.
-
 ### Giraffe
 
 ![Giraffe](./giraffe.jpg)
 
-Show me some chirps.
+#### Assessment
+
+TDD a [kata](../../katas) or feature of your choice.  Keeping in mind the mark scheme below.
 
-- [ ] Writes a failing test before writing production code
+- Writes a failing test before writing production code
   - Required - Several examples of this.
 
-- [ ] Watches the failing test fail for the right reason before writing production code
+- Watches the failing test fail for the right reason before writing production code
   - Required - Several examples of this.
 
-- [ ] Writes only the minimum production code for a test to pass
+- Writes only the minimum production code for a test to pass
   - Required - Several examples of sliming production code, or writing very minimal production code, until necessary to change by future tests. Examples of writing small tests to expose these.
 
-- [ ] Watches the passing test pass before moving onto refactoring
+- Watches the passing test pass before moving onto refactoring
   - Required - Several examples of this.
 
-- [ ] Refactors tests
+- Refactors tests
   - Required - One example of refactoring tests, describing why they are.
 
-- [ ] Refactors production code
+- Refactors production code
   - Required - One example of refactoring production code.
   - Can write tests which follow the AAAT (Arrange, Act, Assert, Teardown)
   - Required - Demonstration of setting up some state first, making an action, and asserting on that action. Teardown is optional if it is not required by the test suite (e.g. RSpec).
 
-- [ ] Can describe expected behaviours from the perspective of the user.
+- Can describe expected behaviours from the perspective of the user.
   - Required - Can write test names which describe the behaviour of the software from a user’s perspective.
 
+- Can describe the situations where using triangulation helps improve test quality.
+  - Required - Can explain what the potential flaws of not including triangulation are.
+  - Required - Can explain how triangulation overcomes these flaws.
+
+- Can demonstrate triangulation when TDDing an example piece of code.
+  - Required - Production code cannot be easily slimed, due to the use of triangulation.
```

Comment:
This reads weirdly.

Reply:
`Production code cannot be easily slimed` - should probs link to the practice of generalising code as tests become more specific

Reply:
Please suggest a change.

Reply:
`- Required - Production code is generalised only when there are failing tests that require generalised production code.`